package.json

```diff
@@ -3,14 +3,16 @@
     "version": "3.3.0",
     "description": "Collection of libraries to create an input mask which ensures that user types value according to predefined format",
     "keywords": [
```

Comment:
Root `package.json` is not used for any publishable library.
It means that property `keywords` does not make sense for it 😄 

Let's remove it totally for this file.